porch/func/go.mod

```diff
@@ -3,33 +3,60 @@ module github.com/GoogleContainerTools/kpt/porch/func
 go 1.17
 
 require (
+	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements v0.0.0-20220316042125-d3eb289008bc
```

Comment:
We can use a released version after we release `apply-replacements`

Reply:
sounds good.